autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -152,6 +167,11 @@ class Input(BlockSchema):
         prompt_values: dict[str, str] = SchemaField(
             advanced=False, default={}, description="Values used to fill in the prompt."
         )
+        max_tokens: int | None = SchemaField(
+            advanced=True,
+            default=None,
+            description="The maximum number of tokens to generate in the chat completion.",
+        )
```

Comment:
Maybe make it more clear that this is for *output* tokens;
```suggestion
        max_output_tokens: int | None = SchemaField(
            advanced=True,
            default=None,
            description="The maximum number of tokens to generate in the chat completion.",
        )
```

Reply:
I don't want to make this clear because each provider is also intentionally unclear on their documentation. 

Open AI:
```
max_completion_tokens: An upper bound for the number of tokens that can be generated for a completion,
              including visible output tokens and
              [reasoning tokens](https://platform.openai.com/docs/guides/reasoning).
```

Claude:
```
          max_tokens: The maximum number of tokens to generate before stopping.

              Note that our models may stop _before_ reaching this maximum. This parameter
              only specifies the absolute maximum number of tokens to generate.

              Different models have different maximum values for this parameter. See
              [models](https://docs.anthropic.com/claude/docs/models-overview) for details.
```

Groq:
```
          max_tokens: The maximum number of tokens that can be generated in the chat completion. The
              total length of input tokens and generated tokens is limited by the model's
              context length.
```

They seem to be able to include input / output without violating the description at all. So I'll keep the vagueness :p